src/util/isObjectEmpty.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Function that accepts object as parameter and checks whether
+ * Object is empty or not
+ * @param {object} obj: the object that is checked to see whether it has
```

Comment:
1. `{object}` is no a valid type
2. `Function that accepts object` is already described by the params
3. a more concise description could be: `Checks whether an Object has no props`